ModList/UI/Markdig/UnityRenderer.cs

```diff
@@ -335,19 +363,18 @@ private IEnumerable<RectTransform> RenderListItem(ListItemBlock item, bool isLoo
                 }
                 else
                 {
-                    if (block is ParagraphBlock para && para.Inline != null)
-                        children = RenderInline(para.Inline, ParagraphFontSize);
-                    else
-                        children = RenderBlock(block);
+                    children = block is ParagraphBlock {Inline: { }} para ? RenderInline(para.Inline, ParagraphFontSize) : RenderBlock(block);
```

Comment:
Spaces inside braces please :)

Reply:
.editorconfig has been adjusted for that 😄